webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import URI from 'urijs';
+import { Spinner } from 'patternfly-react';
+import Table from '../common/table';
+import { STATUS } from '../../constants';
+import './ModelsTable.css';
+
+class ModelsTable extends React.Component {
+  componentDidMount() {
+    this.props.updateSearch(this.props.data);
+  }
+  render() {
+    const {
+      schema,
+      perPage,
+      page,
+      searchQuery,
+      results,
+      sortBy,
+      sortOrder,
+    } = this.props;
+    return this.props.status === STATUS.PENDING ? (
+      <div className="tableloader">
+        <Spinner size="lg" loading />
+      </div>
```

Comment:
Could you move it into a new micro component?
Because the css is only relevant to this component, it can also move into this component and you may not need a ModelsTable.scss

Reply:
@waldenraines made a `LoadingState` component already in kattelo under `move_to_pf`